docs/jwst/tweakreg/README.rst

```diff
@@ -191,7 +191,19 @@ Parameters used for absolute astrometry using a reference catalog:
 * ``sigma``: A positive `float` indicating the clipping limit, in sigma units,
   used when performing fit. (Default=3.0)
 
-Parameters used for absolute astrometry using a reference catalog:
+**Absolute Astrometric fitting parameters:**
```

Comment:
There are several other `abs_xxxx` params described above. Why aren't they in this section that's dedicated to absolute fitting?